roles/nut/templates/upsd.users.j2

```diff
@@ -0,0 +1,12 @@
+# {{ ansible_managed }}
+
+[admin]
+	password = {{ admin_pass }}
+	upsmon master
+	actions = SET
+	instcmds = ALL
+
+[nutcli]
+    password = {{ nutcli_pass }}
+    actions = FSD
+    upsmon master
```

Comment:
Maybe choose one indentation kind ^^ (the `ups.conf` file you pushed uses tabs)

Reply:
Resolved with [fefab0a](https://github.com/rezoleo/ansible-playbooks/pull/6/commits/fefab0ae88a104176dabcfbb748fd0f79651b91a)